lang/es_ES.json

```diff
@@ -184,47 +184,47 @@
     "Copyright": "Derechos de autor",
-    "Our web pages and their contents are subject to copyright law. Unless expressly permitted by law, every form of utilizing, reproducing or processing works subject to copyright protection on our web pages requires the prior consent of the respective owner of the rights. Individual reproductions of a work are allowed only for private use. The company names, product names, service names and logotypes on this web site are for identification purposes only. All trademarks and registered trademarks are the property of their respective owners.": "Nuestras páginas web y sus contenidos están sujetos a la ley de derechos de autor. A menos que expresamente permitido por la ley, toda forma de utilizar, reproducir o procesar obras sujetas a protección por derechos de autor en nuestras páginas web requiere el consentimiento previo del respectivo propietario de los derechos. Las reproducciones individuales de una obra solo están permitidas para uso privado. Los nombres de empresas, nombres de productos, nombres de servicios y logotipos en este sitio web son solo para fines de identificación. Todas las marcas comerciales y marcas registradas son propiedad de sus respectivos propietarios.",
+    "Our web pages and their contents are subject to copyright law. Unless expressly permitted by law, every form of utilizing, reproducing or processing works subject to copyright protection on our web pages requires the prior consent of the respective owner of the rights. Individual reproductions of a work are allowed only for private use. The company names, product names, service names and logotypes on this web site are for identification purposes only. All trademarks and registered trademarks are the property of their respective owners.": "Nuestras páginas web y sus contenidos están sujetos a la ley de derechos de autor. A menos que la ley lo permita expresamente, cualquier tipo de uso, reproducción o procesamiento sujeto a protección por derechos de autor en nuestras páginas web requiere el consentimiento previo del respectivo propietario de los derechos. Las reproducciones individuales de una obra solo están permitidas para uso privado. Los nombres de empresas, nombres de productos, nombres de servicios y logotipos en este sitio web son solo para fines de identificación. Todas las marcas comerciales y marcas registradas son propiedad de sus respectivos propietarios.",
```

Comment:
English text remains

Reply:
![image](https://github.com/user-attachments/assets/334ec9fd-f653-4c62-9ffb-1e12d8a34e0d)

I don't got it.